datafusion/common/src/config.rs

```diff
@@ -264,6 +264,11 @@ config_namespace! {
        /// Default is false.
         pub map_varchar_to_utf8view: bool, default = true
 
+        /// If true, `CHAR` and `Text` and `String` is mapped to `Utf8View` during SQL planning.
+        /// If false, `CHAR` and `Text` and `String` is mapped to `Utf8` during SQL planning.
+        /// Default is true.
+        pub map_char_to_utf8view: bool, default = true
+
```

Comment:
Do we need the two configs? I mean, if we can directly use one.

Imagine that one is true, and another is false, does it make sense? Before we didn't have `utf8view`, varchar,char, text, string all will be utf8, they're consistent.

Reply:
> How about:
> 
> ```
> /// If true, string types (VARCHAR, CHAR, Text, and String) are mapped to `Utf8View` during SQL planning.
> /// If false, they are mapped to `Utf8`.
> /// Default is true.
> pub map_string_types_to_utf8view: bool, default = true
> ```
> 
> Also cc @alamb

Thanks @xudong963 good to me.

Reply:
Addressed in latest PR, thanks!